### DISCUSSION TITLE
Cabling question when using Octal cable

### DISCUSSION THREAD

**Discussion by mahmoudkhalil2** [netbox-community/netbox#13121] [Q&A]
Hello All, now we have cisco device ISR2901 and this device have 4 module slots and we have the below modules installed:
one WIC-1B-S/T-V3 module in slot EHWIC 0
one HWIC-16A module in slot EHWIC 1 

for example: HWIC-16A module has two physical ports and we using OCTAL cable connected to one of this module port, and this octal cable provide us 8 cables connected to 8 devices console ports.

my question, how i can add this cabling topology in the Netbox using OCTAL cable, I mean how can add OCTAL cable in Netbox and connect it's first end to the ISR2901 in module HWIC-16A slot and the other 8 octal cable 8 ends to my 8 devices console ports, because i didn't find the octal cable in the netbox.

I see this old thread but I asked in case this is applicable now in the new Netbox versions
https://github.com/netbox-community/netbox/issues/3018

**Comment by candlerb** [comment]
In Netbox since [v3.3](https://github.com/netbox-community/netbox/releases/v3.3.0), a cable can have multiple terminations at each end. There don't have to be equal numbers at each end, nor do all the terminations at one end have to be into the same device.

Therefore, you just create a single cable with one end connected to your "console server port", and the other end connected to "console ports" on 8 other devices (or to 8 frontports in a patchpanel, or whatever). Job done.

Beware that this is a very simplistic model. The console server still has a single "port", and there's no way of distinguishing the 8 internal console server ports nor associating them individually with the 8 cable legs.

There is also a possibility of confusion in the UI when adding the cable endpoints if all the devices have a console port with the same name: see #10841 (which talks about interfaces, but the same will apply to console ports). This might be better in v3.5, but I haven't retested this use case specifically.

**Reply by mahmoudkhalil2** [reply]
Thanks @candlerb  for your reply, please let me know which interface type from Netbox can be used for the above modules interface because i don't find ISDN BRI ST interface type in netbox match the WIC-1B-S/T-V3 module interface and also don't find Asynchronous interface in netbox match the HWIC-16A module interface.

**Reply by candlerb** [reply]
I think you want a "console server port" type rather than "interface" type (for the multi-port async serial).

For ISDN interface and for a multi-port console server port I'd just choose "Other" as the type.
